### PR TITLE
IL/PS: Updated geometries

### LINKED ISSUE #1473: Update geometries in and around Israel
More specifically:

https://spelunker.whosonfirst.org/id/85632315/
https://spelunker.whosonfirst.org/id/85633739/
https://spelunker.whosonfirst.org/id/85681427/
https://spelunker.whosonfirst.org/id/1159339487/
https://spelunker.whosonfirst.org/id/85681213/
https://spelunker.whosonfirst.org/id/1159339485/

As well as any/all descendant geometries.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1473.

To do:
- [x] Update disputed geometries
- [x] Create new disputed records, as needed
- [x] Update country geometries
- [x] Update region geometries
- [x] Update county geometries
- [x] Create locality geometries for ten+ largest cities in Israel
- [x] Move remaining locality points within new locality polygons to neighbourhood placetype
- [x] De-dup and fix Jerusalem and surrounding area localities
- [x] Create new neighbourhoods in Jerusalem, based on existing locality points and polygons
- [x] Adjust hierarchies of new disputed geometries to include Palestine and Israel, use controlled prop to set
- [x] PIP to update hierarchies
- [x] Validate all geometries and validate GeoJSON

### CODE REVIEW

**Comment by stepps00** [comment]
fbf4e33d157a2db5ff6e46eb3b11e8ba2e4f8ef1: move leftover locality points to neighbourhood placetype
  - Moves any locality point that falls within a new locality polygon to a new neighbourhood record
  - Deprecates existing locality point record, includes superseding work

5c1724001e672374ea10cac3bc240ed7fc2b3ecb: add polygon geometries to large cities in Israel, minus Jerusalem
  - Adds new polygon geometries for the largest cities in Israel
  - Does not include Jerusalem (in separate commit)
  - Also includes smaller cities surrounding Tel Aviv

23e0651239a8a33296c403fea39cc7d6c1802cf7: take care of any duplicate locality points in three cases
  - Follow-up work to take care of superseding duplicate locality points within the new locality polygons (three cases)

fd6c4d13d840f15877b8596148441a2439479a33: update country geoms, move exiting to alt files
  - Updates country geometries of Israel, Palestine, Jordan, Syria, and Lebanon
  - All country geometries now snap to one another
  - Note: This commit adjusts the boundaries of Israel/Palestine, adjusting Israel geometry to include disputed areas
  - Stores existing geometries in alt files

54c94f828ef5e1ea88e8d376ab51037219151d42: update region geometries, store alt geoms
  - Adjusts region geometries 
  - Saves existing geometries in alt files

24013cdb52d71c0ed01fc40ed90671fa13ae5ea5: new disputed geometries, moving existing geoms to alt files
  - Adjusts disputed geometries 
  - Saves existing geometries in alt files

37cf972da6896e92c29a3f3729207c395bf6e87d: new disputed records for Mt Scopus and East Jerusalem
  - Adds two new disputed records, one for East Jerusalem, one for Mt Scopus
  - These are whosonfirst-sourced geometries, but verified in the US State Dept admin file

33ba4c96460066d12e587d8a17ca46280ad85bf4: update county geometries, store alt geoms
  - Adjusts county geometries 
  - Saves existing geometries in alt files
  - Removes overlaps that currently exist
  - Snaps to other placetypes

7486102717471ef55c45cbe339bf7a8acaec55d4: new Jerusalem geometry, save existing as alt file
  - Updates the geometry for the city of Jerusalem
  - Saves existing geometry in an alt file

440d4ee08b0c19762322365a96fa09afb3c241cf: move existing locality points in new Jerusalem polygon to neighbourhood
  - Moves existing locality point records within the new geometry of Jerusalem to the neighbourhood placetype

**Comment by nvkelso** [comment]
We looked at this on screen last week, and I spot checked commits in this PR. They look good.

Looks like you have some final PIP work to take care of.

And then let's discuss options for Jerusalem – right now it's newly one big locality polygon (which reflects defacto situation). We also have the disputed polygon for "East" Jerusalem. So I think downstream users can mix and match these to their politics / compliance. But alternatively we could split the locality polygon up as well and perhaps play with the localized names.

File: data/137/689/141/1/1376891411.geojson [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "iso:country":"XX",
+    "lbl:latitude":31.820219,
+    "lbl:longitude":35.235139,
+    "mps:latitude":31.820219,
+    "mps:longitude":35.235139,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:eng_x_preferred":[
+        "East Jerusalem"
+    ],
```
Comment (nvkelso) [inline data/137/689/141/1/1376891411.geojson]: Please also add Arabic and Hebrew translations, and add `Jerusalem` as an eng_x_alterate name.

**Comment by nvkelso** [comment]
Please also add hierarchy labels masks on the new disputed areas, with same setup as existing disputed areas between these countries.

See comment about East Jerusalem – just the disputed area name is fine.

**Comment by nvkelso** [comment]
For Jerusalem (wof:id = 421168681), please also ensure the English preferred name is Jerusalem, with both `East Jerusalem` and `West Jerusalem` as alternates, and also `al-Quds` to capture the anglicised version of the Arabic. Then verify the default Hebrew name is `יְרוּשָׁלַיִם‬` and the default Arabic name is `القُدس‎`.

File: data/137/689/141/1/1376891411.geojson (hunk 1 above)
Comment (stepps00) [reply data/137/689/141/1/1376891411.geojson]: Fixed here: https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/stepps00/israel-palestine/data/137/689/141/1/1376891411.geojson

**Comment by stepps00** [comment]
https://github.com/whosonfirst-data/whosonfirst-data/pull/1476#issuecomment-462458631 and https://github.com/whosonfirst-data/whosonfirst-data/pull/1476#issuecomment-462456415 have been taken care of in recent commits. 

An upcoming commit will include all hierarchy updates for records in Israel and Palestine. Once approved, I'll validate changes and merge.

**Comment by stepps00** [comment]
[456e3c1](/whosonfirst-data/whosonfirst-data/pull/1476/commits/456e3c196815a257ad15ba8fda64ac96e0a6d8ea) includes PIP work for this branch, over to @nvkelso for review.

**Comment by stepps00** [comment]
Once I validate records and QA a few records, I'll merge.

**Comment by stepps00** [comment]
` [05cc0fb](/whosonfirst-data/whosonfirst-data/pull/1476/commits/05cc0fb5cb74184c434dc0e2c23986cdbd40f0c7)` ensures that all properties are valid using the `wof-validate-index` tool and all geometries are valid using `osgeo`.

**Comment by stepps00** [comment]
QA'd, changes look as expected. There is additional work to do at the locality level, specifically with duplicate features, but we can track that through https://github.com/whosonfirst-data/whosonfirst-data/issues/1266. Merging and closing.

**Pull request merged**